### PR TITLE
DCS-1516 moving remaining service layer tests over to use test data factory

### PR DESCRIPTION
**Pull request by andrewrlee**

### CODE REVIEW

File: server/services/imprisonmentStatusesService.test.ts [hunk 1]
```diff
@@ -77,40 +51,14 @@ describe('Imprisonment statuses service', () => {
...
+        const imprisonmentStatus = imprisonmentStatuses.find(s => s.code === 'convicted-unsentenced')
 
         const result = await service.getImprisonmentStatus('convicted-unsentenced')
 
         expect(hmppsAuthClient.getSystemClientToken).toBeCalled()
         expect(welcomeClient.getImprisonmentStatuses).toBeCalled()
         expect(result).toStrictEqual(imprisonmentStatus)
       })
-
-      it('should return imprisonment status with multiple movement reasons', async () => {
```
Comment (andrewrlee) [inline server/services/imprisonmentStatusesService.test.ts:94]: removed this as this wasn't exercising any additional logic

File: server/services/temporaryAbsencesService.test.ts [hunk 2]
```diff
@@ -15,90 +15,29 @@ describe('Temporary absences service', () => {
 
   const WelcomeClientFactory = jest.fn()
 
-  const temporaryAbsences: TemporaryAbsence[] = [
```
Comment (andrewrlee) [inline server/services/temporaryAbsencesService.test.ts:18]: moving test specific setup into relevant describe blocks

File: server/services/temporaryAbsencesService.test.ts [hunk 3]
```diff
@@ -15,90 +15,29 @@ describe('Temporary absences service', () => {
...
-        {
-          firstName: 'Barry',
-          lastName: 'Smith',
-          dateOfBirth: '1970-01-01',
-          prisonNumber: 'G0012HK',
-          reasonForAbsence: 'External visit',
-          movementDateTime: '2022-01-16T12:30:00',
-        },
-      ])
+      expect(result).toStrictEqual([ant, bat, cat, dog])
```
Comment (GurnankCheema) [inline server/services/temporaryAbsencesService.test.ts:40]: hmmm. Not sure if this helps in understanding what's going on 🤔

**Review by GurnankCheema — APPROVED** [review]
LGTM

**Pull request merged**